src/generic-container/generic-container.ts

```diff
@@ -366,4 +368,9 @@ export class GenericContainer implements TestContainer {
     }
     return this.tarToCopy;
   }
+
+  public withVolumes(...volumes: Volume[]): this {
```

Comment:
This may be more intuitive without using varargs, i.e:

```javascript
.withVolumes(
  { name: "vol1" },
  { name: "vol2" },
)
```

Is less intuitive than:

```javascript
.withVolumes([
  { name: "vol1" },
  { name: "vol2" },
])
```